src/platform-implementation-js/dom-driver/gmail/views/gmail-app-sidebar-view/index.ts

```diff
@@ -9,106 +9,70 @@ import type GmailDriver from '../../gmail-driver';
-    // restricted to being used for references to DOM nodes. When
-    // GmailAppSidebarView is instantiated, we check the element for an
-    // attribute to see whether a previous extension's GmailAppSidebarView has
-    // already set up the sidebar or not.
+    this.#driver = driver;
+    /**
+     * We need to be able to cooperate with other apps/extensions that are
+     * sharing the app sidebar. We store some properties as attributes in the
+     * shared DOM instead of as class properties; class properties are mostly
+     * restricted to being used for references to DOM nodes. When
+     * GmailAppSidebarView is instantiated, we check the element for an
+     * attribute to see whether a previous extension's GmailAppSidebarView has
+     * already set up the sidebar or not.
+     */
     const instanceId = companionSidebarContentContainerElement.getAttribute(
       'data-sdk-sidebar-instance-id',
     );
 
     if (instanceId != null) {
-      this._instanceId = instanceId;
+      this.#instanceId = instanceId;
     } else {
       let primary = new GmailAppSidebarPrimary(
         driver,
         companionSidebarContentContainerElement,
       );
-      this._instanceId = primary.getInstanceId();
+      this.#instanceId = primary.getInstanceId();
       // hot reloading support. Not perfect; this will break any existing panels.
       updates.changes().onValue(() => {
         primary.destroy();
         primary = new GmailAppSidebarPrimary(
           driver,
           companionSidebarContentContainerElement,
         );
-        this._instanceId = primary.getInstanceId();
+        this.#instanceId = primary.getInstanceId();
       });
     }
   }
 
   destroy() {
-    this._stopper.destroy();
+    this.#stopper.destroy();
   }
 
   getStopper() {
-    return this._stopper;
-  }
-
-  _getShouldThreadAppSidebarOpen(): boolean {
-    return (
-      global.localStorage.getItem(
-        'inboxsdk__thread_app_sidebar_should_open',
-      ) !== 'false'
-    );
-  }
-
-  _setShouldThreadAppSidebarOpen(open: boolean) {
-    try {
-      global.localStorage.setItem(
-        'inboxsdk__thread_app_sidebar_should_open',
-        String(open),
-      );
-    } catch (err) {
-      console.error('error saving', err);
-    }
-  }
-
-  // This value controls whether the app sidebar should automatically open
-  // itself when available when the chat sidebar isn't present. It's only set
-  // if the user interacts with the app sidebar button.
-  _getShouldGlobalAppSidebarOpen(): boolean {
-    return (
-      global.localStorage.getItem(
-        'inboxsdk__global_app_sidebar_should_open',
-      ) === 'true'
-    );
-  }
-
-  _setShouldGlobalAppSidebarOpen(open: boolean) {
-    try {
-      global.localStorage.setItem(
-        'inboxsdk__global_app_sidebar_should_open',
-        String(open),
-      );
-    } catch (err) {
-      console.error('error saving', err);
-    }
```

Comment:
This class isn't part of public interface.

Based on grepping, it appears that this code was moved into src/platform-implementation-js/dom-driver/gmail/views/gmail-app-sidebar-view/primary/index.tsx.